### DISCUSSION TITLE
MouseOver Text for all (or at least some) of dashboard elements

### DISCUSSION THREAD

**Discussion by FrankJaco** [home-assistant/frontend#17896] [Dashboard]
As more and more of my home is being controlled by Home Assistant, I have more and more buttons and controls on my dashboards. Despite trying to automate things as many things as possible, local control is also required. Especially on my smartphone where the screen is small, I rely on buttons with icons only to best use the space. I would like to suggest that Dashboard elements include a MouseOver text field.   For example I have several recessed light fixtures using the "mdi:light-recessed" icon. I currently rely on the position of the button on the screen and the organization of what room it is to know which light I am turning on.  At times my family forgets (as do I).  It would be a great feature that if I simply hovered over the button and MouseOver text would appear providing more info regarding the element. E.g. "Garage Entry Light". etc. etc.

**Comment by karwosts** [comment]
How does mouseover even work on a smartphone though? I thought that was maybe a problem with tooltip-only accessible information is there's no real way to "hover" on a smartphone without pressing the button.
